src/main/java/uk/gov/justice/hmpps/prison/api/model/LatestTusedData.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.hmpps.prison.api.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDate
+
+@Schema(description = "Latest TUSED dates and associated info for offender")
+data class LatestTusedData(
+  val latestTused: LocalDate?,
+  val latestOverrideTused: LocalDate?,
+  val comment: String?,
+  val offenderNo: String?,
```

Comment:
don't think this needs to be nullable?

Reply:
I think it needs to be nullable so it can be deserialized correctly